mailmerge/api.py

```diff
@@ -94,6 +91,39 @@ def addattachments(message, template_path):
         original_text = message.get_payload()
         multipart_message.attach(email.mime.text.MIMEText(original_text))
         message = multipart_message
+    # HACK: For Python2 (see comments in `_create_boundary`)
+    message = _create_boundary(message)
+    return message
+
+
+def convert_markdown(message):
+    """Convert markdown in message text to HTML."""
+    if not message['Content-Type'].startswith("text/markdown"):
```

Comment:
I think the code would be more readable if the `if` statement was in the calling function.  Then, put an `assert` right here.

Reply:
Resolved in 3b7253653df27ce162198612c2c6d5f1ce2ccab4.